addons/hr_timesheet/models/project.py

```diff
@@ -207,7 +207,7 @@ def action_timer_start(self):
     def action_timer_stop(self):
         # timer was either running or paused
         if self.user_timer_id.timer_start and self.display_timesheet_timer:
-            minutes_spent = super().action_timer_stop()
+            minutes_spent = self.user_timer_id._get_minutes_spent()
```

Comment:
Congratulation ! The action named `action_timer_stop` no longer stops the timer :rocket: 
You should probably pause the timer when the model opens, then resume it if it's cancelled